hardhat.config.ts

```diff
@@ -60,25 +52,30 @@ const config: HardhatUserConfig = {
   networks: {
     mainnet: {
       chainId: 1,
-      url: process.env.MAINNET_PROVIDER || "",
-      accounts: process.env.MAINNET_DEPLOYER_PRIVATE_KEY ? [process.env.MAINNET_DEPLOYER_PRIVATE_KEY] : [],
+      url: process.env.MAINNET_PROVIDER,
+      accounts: [process.env.MAINNET_DEPLOYER_PRIVATE_KEY || ""],
```

Comment:
@christopherdancy -- was this file reverted for a reason? I had updated this file so the contracts could be run locally without needing the private keys (having a blank private key here throws an error)